letsencrypt/storage.py

```diff
@@ -127,6 +122,59 @@ def get_link_target(link):
     return os.path.abspath(target)
 
 
+def _relevant(option):
+    """
+    Is this option one that could be restored for future renewal purposes?
+    :param str option: the name of the option
+
+    :rtype: bool
+    """
+    # The list() here produces a list of the plugin names as strings.
+    from letsencrypt import cli
```

Comment:
Python will import module only once independently of the number or `import module`. To reimport a module, one explicitly must call `reload(module)`.
But it is considered bad practice to not do import at the top of the file: https://www.python.org/dev/peps/pep-0008/#imports.